CONTRIBUTING.md

```diff
@@ -44,14 +49,11 @@ cd frontend
 # Symlink dev environment on unix
 ln -hfs .env.local.example .env.local
 
-# Symlink dev environment on Windows
+# Symlink dev environment on Windows (CMD. run as admin)
 mklink .env.local .env.local.example
 
-# Install dependencies
-yarn
-```
-
-### NOTE! For existing clones update github remote to podkrepi-bg/frontend
+cal .env.local.example
+ng clones update github remote to podkrepi-bg/frontend
```

Comment:
Isn't `ng` an Angular command? Why is it needed for this project where we aren't using Angular at all?

Reply:
We also miss the `yarn install` command